src/Infrastructure/Process/ProcessFactory.php

```diff
@@ -15,10 +16,14 @@ class ProcessFactory
      *
      * @return \Symfony\Component\Process\Process<\Generator>
      *
-     * @throws \Symfony\Component\Process\Exception\LogicException
+     * @throws \PhpTuf\ComposerStager\Exception\LogicException
      */
     public function create(array $array, ...$args): Process
     {
-        return new Process($array, ...$args);
+        try {
+            return new Process($array, ...$args);
+        } catch (\Symfony\Component\Process\Exception\LogicException $e) { // @codeCoverageIgnore
+            throw new LogicException($e->getMessage(), $e->getCode(), $e->getPrevious()); // @codeCoverageIgnore
```

Comment:
Oops! Good catch on passing `$e->getPrevious()`. Fixed.

The purpose of wrapping the (Symfony) exception (with our own) is to prevent the domain layer above from depending on third party code--which is the responsibility of the infrastructure layer.